core/src/main/java/org/opensearch/sql/ast/dsl/AstDSL.java

```diff
@@ -249,10 +249,10 @@ public static Function function(String funcName, UnresolvedExpression... funcArg
   }
 
   /**
-   * CASE
-   *     WHEN search_condition THEN result_expr
-   *     [WHEN search_condition THEN result_expr] ...
-   *     [ELSE result_expr]
+   * CASE<br>
```

Comment:
This file already included in #312 (part 1)

Reply:
Great catch! Entire `**/ast` directory was included in both. I removed it from this repo and will use it from PR #1.